runtime/src/main/java/org/corfudb/runtime/GarbageInformer.java

```diff
@@ -22,6 +55,26 @@ public synchronized void add(UUID streamId, long detectorAddress, SMREntry.SMREn
         }
     }
 
+    public void gc() {
+        garbageQueue.addAll(addressToGarbage.entrySet());
```

Comment:
Let's discuss this v.s. add directly to the queue and taking a batch

Reply:
Okay. Let discuss it offline.